lib/datadog/ci/transport/api/ci_intake.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative "base"
+require_relative "../http"
+
+module Datadog
+  module CI
+    module Transport
+      module Api
+        class CIIntake < Base
```

Comment:
`CIIntake` 😛

Reply:
Looks ugly, I know :( EVPProxy looks not so much better
I think I should drop abbreviations and call them CiIntake and EvpProxy

Reply:
The official name of the intake is CiTestCycle, I will use it